container-scan/action.yaml

```diff
@@ -1,152 +1,78 @@
-name: "Security Scan Composite Action"
-description: "GitHub Action for scanning container image for vulnerabilities using Lacework"
+name: "Container Security Scan"
+description: "Scans container images for vulnerabilities using Lacework"
 
 inputs:
-  dockerhub-user:
+  image-name:
     required: false
-    default: ""
-    description: username for dockerhub
-  dockerhub-password:
+    description: "Docker image name"
```

Comment:
You don't need quotes on all of these values after description, as they are strings by default